File: js/day81.js

const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

let cardNumber = "";
rl.question('Please input your card number: ', (answer) => {
  cardNumber = answer;
  

    if (cardNumber.match(/^\d{16}$/)) {
    console.log("Your card number is valid"); 
    }
    else {
        console.error("Your card number is invalid!!!!");
    }
    rl.close(); 
});

File: cSharp/day81.cs
using System;
using System.Text.RegularExpressions;


Console.WriteLine("Please input your card number:");


string cardNumber1 = Console.ReadLine();

Regex cardPattern1 = new Regex("^\\d{16}$");

if (cardPattern1.Match(cardNumber1).Success) {
    Console.WriteLine("Valid card number");
}
else {
    Console.WriteLine("Invalid card number");
}
